src/components/nora-textfield.tsx

```diff
@@ -0,0 +1,163 @@
+import React, { useState } from "react";
+import { TextFieldProps } from "@mui/material";
+import styled from "@emotion/styled";
+
+// importing "TextField" from MUI is cheating ;) You're supposed to recreate it not use it.
+// if you inspect the HTML on their docs you can see that they use a <textarea> for multiline
+// and <input> for single line. I changed your code
+const StyledTextArea = styled.textarea`
+  font-family: Roboto, Helvetica, Arial, sans-serif;
+  font-size: 16px;
+  line-height: 23px;
+  text-align: left;
+  text-indent: 0px;
+  text-rendering: auto;
+  box-sizing: border-box;
+  display: block;
+  border: 0;
+  height: 56px;
+
+  &:focus {
+    writing-mode: horizontal-tb !important;
+  }
+
+  &.textfield__variant--outlined {
+    padding: 16.5px 14px;
+    border-radius: 4px;
+    border-color: #0000003b;
+    border-style: solid;
+    border-width: 1px;
+    width: 195px;
+  }
+
+  &.textfield__variant--filled {
+    position: relative;
+    background-color: #0000000f;
+    border-top-left-radius: 4px;
+    border-top-right-radius: 4px;
+    width: 167px;
+    height: 23px;
+    box-sizing: content-box;
+    padding: 25px 12px 8px 12px;
+
+    &:hover {
+      background-color: #00000017;
+    }
+
+    &:focus {
+      outline: none;
+      border-bottom: 2px solid #1976d2;
```

Comment:
So here is your issue. When you focus you add a border which makes the element 2px taller. I would always have a border, but with `color: transparent`, and then just change the color on focus